meta-resin-jethro/classes/bash-completion.bbclass

```diff
@@ -0,0 +1,7 @@
+DEPENDS_append_class-target = " bash-completion"
```

Comment:
If we do this, i.e. copy the entire class here, we can get rid of this: https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%25.bbappend#L18

Reply:
The above should be turned into an issue so it gets done in a different PR if not in the current PR.

Reply:
https://github.com/resin-os/meta-resin/issues/833